.github/actions/check_cirrus_cron/test.sh

```diff
@@ -74,11 +74,23 @@ expect_regex \
 
 #####
 
+msg "$header make_email_body.sh name and link"
+# Job names may contain spaces, confirm lines are parsed properly
+echo -e '1234567890 cirrus-cron test job' >> "$ID_NAME_FILEPATH"  # blank lines should be ignored
+$base/make_email_body.sh
+expected="Cron build 'cirrus-cron test job' Failed: https://cirrus-ci.com/build/1234567890"
+if ! grep -q "$expected" ./artifacts/email_body.txt; then
+    die "Expecting to find string '$expected' in generated e-mail body:
+$(./artifacts/email_body.txt)"
```

Comment:
Missing less-than? `$(< ./artifacts...)`

Reply:
ya, and there's some bug somewhere with `.` being the "wrong" directory.  Pushing a fix for the redirect and explicitly setting `.` as `$GITHUB_WORKSPACE` everywhere this was assumed before.